DTS-STN/Service-Canada-Design-System
# updating Footer Id names

### Footer Component

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @zeddotes

#### Description & Background Context:
Id on the footer have the attribute "[object Object]0" which is invalid. the change adds a constant value, and the index, to provide a valid attribute. 

#### Where should the reviewer start?

File name: Footer.js

#### How should this be tested? 
Use storybook on the current released version, open developer tools and inspect the Id of the landscape links, you will see the values above "[object Object]0" for the Contact Us link on the landscape area. 
![image](https://user-images.githubusercontent.com/52539578/189189432-14ac4efc-45eb-4dce-b461-3a1dc77509c9.png)

Run the fix and verify using storybook, verify the Contact Us link has the Id "LandscapeLink0" 

#### Are there any breaking changes? 
Not that I see.
